DataTool/ToolLogic/List/Misc/ListWorkshop.cs

```diff
@@ -72,6 +73,18 @@ public class ListWorkshop : JSONTool, ITool {
                             Parameters = ParseParameters(action.m_params)
                         });
                         break;
+                    case STU_ACDD45D0 stu:
+                        STU_611C97A9[] def_assets = new STU_611C97A9[0];
+                        @return.Extensions = stu.m_86397C09.Select(ext => new WorkshopExtension {
+                            DisplayName = GetString(ext.m_displayName),
+                            Description = GetString(ext.m_description),
+                            UnlockedValues = (ext.m_assets == null ? def_assets : ext.m_assets).Select(v => new ExtensionVal {
```

Comment:
why do you need a default? cant you just do `ext.m_assets?.Select`

Reply:
indeed